scripts/mkcloud

```diff
@@ -15,6 +15,18 @@
 # 2. This tool relies on the script qa_crowbarsetup.sh
 # 3. Please 'export' environment variables according to your needs.
 
+# FIXME: separate user-tweakable parameters from script local variables.
+# Currently there is no clearly defined interface point.  One of the
+# causes of this is violation of the common shell coding standard which
+# uses uppercase for environment variables and constants, and lowercase
+# for local variables.
+
+for required_var in cloud; do
+    if [ "${!required_var:+defined}" != 'defined' ]; then
+        echo "mkcloud requires $required_var variable to be set; aborting" >&2
+        exit 1
```

Comment:
-1 "mkcloud requires cloud variable to be set" breaks mkcloud help function as discovered by travis CI test.